### PR TITLE
Changes PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -982,7 +982,7 @@
         "apy": "Juros anual <0/> variável"
       },
       "withdraw": {
-        "title": "Resgatar empréstimo",
+        "title": "Resgatar",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:985]:
```suggestion
        "title": "Resgatar ativo emprestado",
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "spotTrades": "Desculpe, a opção de operar spot está em manutenção, por favor verifique o nosso <0>discord</0> para atualizações.",
     "swapTrades": "Desculpe, a opção de trocar criptos está em manutenção, por favor verifique o nosso <0>discord</0> para atualizações.",
     "depositLend": "Desculpe, a opção de emprestar ativos está em manutenção, por favor verifique o nosso discord (link no rodapé) para atualizações.",
-    "withdrawLend": "Desculpe, a opção de resgatar empréstimos está em manutenção, por favor verifique o nosso discord (link no rodapé) para atualizações.",
+    "withdrawLend": "Desculpe, a opção de resgatar está em manutenção, por favor verifique o nosso discord (link no rodapé) para atualizações.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:35]:
```suggestion
    "withdrawLend": "Desculpe, a opção de resgatar ativo emprestado está em manutenção, por favor verifique o nosso discord (link no rodapé) para atualizações.",
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -1250,6 +1265,10 @@
         "liquidityAmount": "Quantidade",
         "transactionHash": "ID transação",
         "status": "Status"
+     },
+      "txType": {
+        "added": "Adicionar",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1270]:
```suggestion
        "added": "Adicionar liquidez",
```

File: src/locales/pt_br/translation.json [hunk 4]
```diff
@@ -1250,6 +1265,10 @@
...
+        "removed": "Retirar"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1271]:
```suggestion
        "removed": "Retirar liquidez"
```

File: src/locales/pt_br/translation.json [hunk 5]
```diff
@@ -1462,7 +1481,7 @@
     "linkCopied": "Link copiado",
     "referralHistory": "Histórico",
     "terms": "Termos e condições",
-    "text": "Indique a Sovryn para seus amigos e ganhe <1>bônus de recompensa em $SOV e .01% das taxas cobradas</1> em cada transação feita por novas carteiras criadas com o seu link de indicação.",
+    "text": "Indique e ganhe <1>recompensas em $SOV + 0.01% das taxas cobradas pela Sovryn</1> nas transações realizadas pelas carteiras criadas com o seu link de indicação.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1484]:
```suggestion
    "text": "Convide seus amigos e ganhe <1>recompensas em $SOV + 0.01% das taxas cobradas pela Sovryn</1> nas transações realizadas pelas carteiras criadas com o seu link de indicação.",
```

**Comment by soulBit** [comment]
@bitcoinuser can you check that all strings are up to date based on latest `development` branch, and make any changes if needed. Once that is done I will merge this PR

**Comment by bitcoinuser** [comment]
> @bitcoinuser can you check that all strings are up to date based on latest `development` branch, and make any changes if needed. Once that is done I will merge this PR

Yes, all ok. You can merge.

**Pull request merged**